### PR TITLE
XTTS functions

### PR DESCRIPTION
**Pull request by Sylphar**
Added XTTS as an option for TTS. 
Changed requirements in startup.bat and requirements_linux.txt from TTS (old, non maintained) to coqui-tts (the same, but maintained).
Changed readme.md to include my little written tutorial (your call if you want to keep it)

### CODE REVIEW

File: scripts/play_tts.py [hunk 1]
```diff
@@ -2,9 +2,14 @@
 import simpleaudio as sa
 from scripts.utils import HiddenPrints
 import torch
+from TTS.api import TTS
+import numpy as np
```
Comment (Rubiksman78) [inline scripts/play_tts.py:6]: Import not used in this script.

File: startup.bat [hunk 2]
```diff
@@ -29,11 +29,11 @@ if /i "%useSpeech%" EQU "y" (
...
 )
 
-:: Install yourTTS
+:: Install yourTTS and XTTS
 if /i "%installYourTTS%" EQU "y" (
-    echo Installing yourTTS...
+    echo Installing yourTTS and XTTS...
     "libs/python.exe" -m pip install simpleaudio-1.0.4-cp39-cp39-win_amd64.whl
-    "libs/python.exe" -m pip install TTS
+    "libs/python.exe" -m pip install coqui-tss
```
Comment (Rubiksman78) [inline startup.bat:36]: Small typography it should be `coqui-tts`.

**Review by Rubiksman78 — COMMENTED** [review]
Some minor fixes in the scripts + typography in the setup that would make the installation of CoquiAI TTS impossible.

**Review by Sylphar — COMMENTED** [review]
Just added a tk.Label line to add the words "Voice sample" in front of the second voice sample choise. Also renamed the first one "Tortoise voice sample".
Goal is simply to add one label to avoid people bing confused if they relaunch the script after a while
